server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/GlobalQuotaRoutesTest.java

```diff
@@ -272,79 +290,130 @@ public void getQuotaShouldReturnBothWhenValueSpecified() throws Exception {
     }
 
     @Test
-    public void getQuotaShouldReturnBothDefaultValues() throws Exception {
+    public void getQuotaShouldReturnNothingWhenNothingSet() {
         JsonPath jsonPath =
             given()
-                .get(GlobalQuotaRoutes.QUOTA_ENDPOINT)
+                .get("/quota")
             .then()
                 .statusCode(HttpStatus.OK_200)
                 .contentType(ContentType.JSON)
                 .extract()
                 .jsonPath();
 
-        assertThat(jsonPath.getLong("size")).isEqualTo(Quota.UNLIMITED);
-        assertThat(jsonPath.getLong("count")).isEqualTo(Quota.UNLIMITED);
+        assertThat(jsonPath.getObject("size", Long.class)).isNull();
+        assertThat(jsonPath.getObject("count", Long.class)).isNull();
     }
 
     @Test
-    public void getQuotaShouldReturnBothWhenNoCount() throws Exception {
-        int maxStorage = 42;
-        maxQuotaManager.setDefaultMaxStorage(maxStorage);
+    public void getQuotaShouldReturnOnlySizeWhenNoCount() throws Exception {
+        maxQuotaManager.setDefaultMaxStorage(QuotaSize.size(42));
 
         JsonPath jsonPath =
-            given()
-                .get(GlobalQuotaRoutes.QUOTA_ENDPOINT)
+            when()
+                .get("/quota")
             .then()
                 .statusCode(HttpStatus.OK_200)
                 .contentType(ContentType.JSON)
                 .extract()
                 .jsonPath();
 
-        assertThat(jsonPath.getLong("size")).isEqualTo(maxStorage);
-        assertThat(jsonPath.getLong("count")).isEqualTo(Quota.UNLIMITED);
+        assertThat(jsonPath.getLong("size")).isEqualTo(42);
```

Comment:
Assert softly when there is several asserts would be IMO a good improvment...

Reply:
ok